docs/env/ubuntu/20.04.md

```diff
@@ -0,0 +1,88 @@
+# Ubuntu 20.04 Setup Instructions
+
+## Verify Hardware Support
+
+Most modern GPUs support OpenCL. For integrated graphics devices (iGPUs), use `lscpu` to get the processor SKU. Detailed information for Intel SKUs is available from [ark.intel.com](ark.intel.com). Detailed information for AMD processors is available from [AMD's product page](https://www.amd.com/en/products/specifications/processors).
+
+## Build Dependencies
+
+OCL Headers:
+
+```bash
+sudo apt install opencl-c-headers opencl-clhpp-headers
+```
+
+The OpenCL API has its own set of header files; the above command installs both C and C++ headers files. The C header can be found in `<CL/cl.h>`; the C++ header is in `<CL/cl.hpp>`.
```

Comment:
FYI, the C++ binding header file is now `opencl.hpp` in the latest sources on [GitHub](https://github.com/KhronosGroup/OpenCL-CLHPP).  This is a renamed version of the older `cl2.hpp` to make it clearer that it supports all versions of OpenCL and not just OpenCL 2.0 and newer, and replaces the even older `cl.hpp` header.

Reply:
Got it, good to know. As of today on my up-to-date Ubuntu 20.04 system, the package still contain the old cl2.hpp file:

```
caleb@storm:~$ dpkg --listfiles opencl-clhpp-headers
/.
/usr
/usr/include
/usr/include/CL
/usr/include/CL/cl.hpp
/usr/include/CL/cl2.hpp
/usr/share
/usr/share/doc
/usr/share/doc/opencl-clhpp-headers
/usr/share/doc/opencl-clhpp-headers/changelog.Debian.gz
/usr/share/doc/opencl-clhpp-headers/copyright
caleb@storm:~$ 
```
I can't promise my brain will remember to double-check this for the next set of instructions, but I'll try. Please feel free to flag it for correction if you see an issue.